### PR TITLE
Semi Random List Generation

### PR DESCRIPTION
**Pull request by ciliakas**
Added a static method to generate semi random lists, using the options pattern to get hardcoded titles from appsettings and creating some random numbers for the prices. Use example is shown in Entry.cshmtl.cs

### CODE REVIEW

File: src/DataManagement/Entries/EntryList.cs [hunk 1]
```diff
@@ -79,5 +82,28 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        [SuppressMessage("ReSharper", "PossibleNullReferenceException")]
+        public static IEntryList RandomList(IConfiguration configuration, EntryType entryType)
+        {
+            IEntryList list = new EntryList(entryType);
+            var section = configuration.GetSection(entryType.ToString());
```
Comment (Stokalas) [inline src/DataManagement/Entries/EntryList.cs:90]: what is this line ? :D

File: src/Pages/Login.cshtml.cs [hunk 2]
```diff
@@ -30,10 +31,10 @@ public class LoginModel : PageModel
 
         private UserDatabase UserDatabase { get; }
         private EmailSender EmailSender { get; }
-        public LoginModel(UserDatabase userDatabase, EmailSender emailSender)
+        public LoginModel(UserDatabase userDatabase, IOptions<EmailSender> emailSenderSettings)
         {
             UserDatabase = userDatabase;
-            EmailSender = emailSender;
+            EmailSender = emailSenderSettings.Value;
```
Comment (Stokalas) [inline src/Pages/Login.cshtml.cs]: Is this some kind of fix related that it don't works for me in dev? :D

**Review by Stokalas — APPROVED** [review]
Will test it when implementing, seems nice

File: src/Pages/Login.cshtml.cs (hunk 2 above)
Comment (ciliakas) [reply src/Pages/Login.cshtml.cs]: Yeah, I've fixed in dev now as well :D

File: src/DataManagement/Entries/EntryList.cs (hunk 1 above)
Comment (ciliakas) [reply src/DataManagement/Entries/EntryList.cs:90]: The configuration object is used to access the appsettings file, from which we take the section of either 'Income' or 'Expense' and use the values in it to set the names of the entries in the list

**Comment by Stokalas** [comment]
I've checked in dev and now Email thing works, nice ad quick fix! :D

**Comment by Stokalas** [comment]
Also I've tested it inside my exceptions branch and it seems to be working well.

**Pull request merged**